account_credit_control/security/account_security.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data noupdate="0">
+    <record id="category_credit_control" model="ir.module.category">
+        <field name="name">Credit Control</field>
+        <field name="sequence" eval="99" />
+    </record>
+
+    <record id="group_account_credit_control_info" model="res.groups" context="{'noadmin':True}">
+        <field name="name">Info</field>
+        <field name="category_id" ref="category_credit_control"/>
+    </record>
+
+    <record id="group_account_credit_control_user" model="res.groups" context="{'noadmin':True}">
+        <field name="name">User</field>
+        <field name="category_id" ref="category_credit_control"/>
+        <field name="implied_ids" eval="[(4, ref('group_account_credit_control_info'))]"/>
+    </record>
+
+    <record id="group_account_credit_control_manager" model="res.groups">
+        <field name="name">Manager</field>
+        <field name="category_id" ref="category_credit_control"/>
+        <field name="implied_ids" eval="[(4, ref('group_account_credit_control_user'))]"/>
+    </record>
+</data>
+
+<data noupdate="1">
+    <record id="credit_control_run_rule" model="ir.rule">
+        <field name="name">Credit Control Run</field>
+        <field name="model_id" ref="model_credit_control_run"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+    </record>
+    <record id="credit_control_line_rule" model="ir.rule">
+        <field name="name">Credit Control Line</field>
+        <field name="model_id" ref="model_credit_control_line"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+    </record>
+    <record id="credit_control_policy_rule" model="ir.rule">
+        <field name="name">Credit Control Policy</field>
+        <field name="model_id" ref="model_credit_control_policy"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+    </record>
+    <record id="credit_control_analysis_rule" model="ir.rule">
+        <field name="name">Credit Control Analysis</field>
+        <field name="model_id" ref="model_credit_control_analysis"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+    </record>
+    <record id="credit_control_partner_payment_action_type_rule" model="ir.rule">
+        <field name="name">Credit Control Partner Payment Action Type</field>
+        <field name="model_id" ref="model_res_partner_payment_action_type"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
```

Comment:
All record rules should be updated to ['|',('company_id','=',False),('company_id', 'in', company_ids)]

Reply:
Fixed record rules and added guideline in migration guide:

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0/_compare/bc329d0741d5606c7b4d441c64cd46e727da5528...fbcfc75d7f45d3057044c6943df9f4a94c8f652f